core/primitives/src/views.rs

```diff
@@ -1324,8 +1324,10 @@ pub struct EpochValidatorInfo {
     pub current_proposals: Vec<ValidatorStakeView>,
     /// Kickout in the previous epoch
     pub prev_epoch_kickout: Vec<ValidatorKickoutView>,
-    /// Epoch start height
+    /// Epoch start block height
     pub epoch_start_height: BlockHeight,
+    /// Epoch height
+    pub epoch_height: EpochHeight,
```

Comment:
I would not say that `validators` endpoint is the best place to expose this value, but I don't have a better place in mind either.

Reply:
Same to me, expose in block api there's significantly more change (Block and BlockView stores nothing about epoch info, and have to refer back to RuntimeAdapter, but RuntimeAdaptor is in neard). Another more proper option maybe add a EXPERIMENTAL_epoch_info and in future supersede validators since epoch_info also covers everything validators in epoch, plus specific epoch information